addons/hr_holidays/models/hr_employee.py

```diff
@@ -385,9 +385,9 @@ def _get_mandatory_days(self, start_date, end_date):
     @api.model
     def _get_contextual_employee(self):
         ctx = self.env.context
-        if 'employee_id' in ctx:
+        if self.env.context.get('employee_id') is not None:
```

Comment:
Is the `is not none` part necessary? :thinking: since the id cannot be 0, anything else than none will give a truthy value, same below:
```suggestion
        if self.env.context.get('employee_id'):
```

Reply:
The employee_id in the context can be False at a scenario like this commit https://github.com/odoo/odoo/commit/7033b1a535105218ba7bb2406e18ee70d28b8bda so I think it is better to check if it is None.